resource.language.zh_tw/addon.xml

```diff
@@ -6,6 +6,7 @@
   provider-name="Team Kodi">
   <requires>
     <import addon="kodi.resource" version="1.0.0"/>
+    <import addon="resource.language.zh_cn" version="9.0.0"/> <!-- To get his font file about where support all -->
```

Comment:
This is added new to get on `zh_tw` and `ja_jp` the font file from `zh_cn` where contains all.